### ISSUE TITLE
[false-positive] Safely use single quotes instead

### ISSUE THREAD

**Issue by b1rdex** [kalessil/phpinspectionsea#119]
```php
$a = <<<'NOWDOC'
test
NOWDOC;
```

These example should not be advised to change to single quotes, because NOWDOC acts like single quotes already.

And more: HEREDOC with advice to change to single quotes should be converted to NOWDOC instead of just single quotes.

**Comment by kalessil** [comment]
Fixed, thank you for reporting @b1rdex
